b3lb/rest/b3lb/endpoints.py

```diff
@@ -150,14 +149,21 @@ async def create(request, endpoint, params, node, secret):
     except KeyError:
         return HttpResponse(constants.RETURN_STRING_MISSING_MEETING_ID, content_type='test/html')
 
+    try:
+        asset = Asset.objects.get(tenant=secret.tenant)
+    except:
+        asset = None
```

Comment:
This could trigger exceptions later on when trying to access `asset.logo` etc.

A improvement might be to use [`get_or_create`](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#get-or-create) above so there is always an *Asset* instance, although it might have no logo nor slide.

Reply:
`get_or_create` used